code/modules/projectiles/guns/projectile/shotgun/pump.dm

```diff
@@ -56,7 +56,7 @@
 	icon = 'icons/obj/guns/projectile/sawnoff/shotgun.dmi'
 	icon_state = "shotgun"
 	item_state = "shotgun"
-	w_class = ITEM_SIZE_BULKY
+	w_class = ITEM_SIZE_NORMAL
```

Comment:
Why would I ever saw it down again?

Reply:
Had discussion about it - amending this. Grizzly will be reverted to huge n’ left as is.